src/properties/section-properties.ts

```diff
@@ -137,6 +140,17 @@ export function sectionPropertiesFromNode(
 export function sectionPropertiesToNode(data: SectionProperties = {}): Node {
 	return create(
 		`element ${QNS.w}sectPr {
+			if (exists($footnotes)) then element ${QNS.w}footnotePr {
+				element ${QNS.w}numFmt { 
+					attribute ${QNS.w}val { $footnotes('numberingFormat')}
+				},
+				element ${QNS.w}pos { 
+					attribute ${QNS.w}val { $footnotes('position')}
+				},
+				element ${QNS.w}numRestart { 
+					attribute ${QNS.w}val { $footnotes('restart')}
+				}
```

Comment:
All these properties seem to be oiptional, please add nullchecks to avoid errors.